.travis.yml

```
@@ -126,10 +118,20 @@ matrix:
               false
             fi
           done
-    # FFmpeg interation build
-    - name: FFmpeg patch
+   # GCC & Clang builds
```

```suggestion
    # GCC & Clang builds
```

Align with the indentation below